README.md

```diff
@@ -137,6 +140,25 @@ the ArcGIS Feature Service.
 }
 ```
 
+### Query Polylabel
+
+Polylabel data (see: [polylabel](https://github.com/mapbox/polylabel)) is
+provided on the `polylabel` field.
+
+`polylabel` is a longitude, latitude tuple of the most distant, internal point
+from the polygon outline.
```

Comment:
Could this read something like this? Something less technical sounding:

> Polylabel data is provided on the `polylabel` field. A polylabel is the optimal point within a polygon to place a marker provided as a `[x, y]` pair.
> 
> See [Mapbox's official Polylabel documentation][polylabel] for more details.

Reply:
Yep. Revised.

Reply:
I think something happened in the copy/paste here. Can you double check?

Also the link at the end of the doc is in the wrong format. Should be `[name]: url` format.

Reply:
My b on the link format, but the above should be fixed now hopefully, lol